src/components/container/Container.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import {ContainerView} from './Container.view';
+
+export const Container = ({pageContainer, background, isVerticallyCentered = false, horizontalPadding = 28}) =>{
```

Comment:
please set by default horizontalPadding from our `size` object

Reply:
I am a little confused about this part. Is this based on figma?